stock_move_actual_date/models/stock_move.py

```diff
@@ -32,6 +32,13 @@ def _compute_actual_date(self):
                 self.with_context(tz=tz), rec.date
             )
 
+    def _action_done(self, cancel_backorder=False):
+        moves = super()._action_done(cancel_backorder)
+        # i.e. Inventory adjustments with actual date
+        if self._context.get("force_period_date"):
+            self.write({"actual_date": self._context["force_period_date"]})
```

Comment:
This needs to go before `super()`, or the value is not passed to the account move. Apply filtering on `self` as appropriate.

Reply:
We can't go before `super()` because compute method will override the `actual_date` when it updates the date with current datetime during `action_done` process.

Reply:
@AungKoKoLin1997 Pass the `actual_date` context, then? Please suggest a good optimal solution.

Reply:
I thought about using context in compute method but rousseldenis mentioned he doesn't like computed stored fields based on context values. So, community may not accept this approach. Let me find the optimal solution.